bin/citgm-all.js

```diff
@@ -93,6 +103,17 @@ function runCitgm (mod, name, next) {
     return next();
   }
 
+  if ((app.excludeTags && mod.tag && mod.tag.includes(app.excludeTags)) ||
```

Comment:
I think it might make sense to break this out into it's own module / function that can be tested. I'm also not very thrilled about the massive boolean logic, is hard to follow and a bit fragile